README.md

```diff
@@ -182,6 +182,7 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Shelley Vohr ([@codebytere](https://github.com/codebytere))
 - Waleed Ashraf ([@waleedashraf](https://github.com/waleedashraf))
 - Yagiz Nizipli ([@anonrig](https://github.com/anonrig))
+- Paula Paul  ([@paulapaul](https://github.com/paulapaul))
 
```

Comment:
I suggest you move this to another PR and add yourself in alphabetical order. Thanks!

```suggestion

```